docs/source/specs/openapi.json

```diff
@@ -1273,6 +1273,18 @@
               ]
             }
           },
+          {
+            "in": "query",
+            "name": "service_account_client_ids",
+            "required": false,
+            "description": "By specifying a list of client IDs with this query parameter, RBAC will return an object with the specified client ID and it's matching boolean value to flag whether the client ID is present in the group or not",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            }
+          },
```

Comment:
I realised that this new query param is resulting in new response schema, can you please add it? I believe we need it for QE tests cc @vbelchio @mfindra

Reply:
I have updated the specification for this one. I have used `additionalProperties` for the response, although I am not very sure if that's correct or not. Let me know if this breaks any clients.

Reply:
awesome, thank you